### DISCUSSION TITLE
hainanu-course-comments/1.%E5%AD%A6%E9%95%BF%E5%AD%A6%E5%A7%90%E8%AF%B4/%E6%B5%B7%E5%8D%97%E5%A4%A7%E5%AD%A6%E7%BE%8E%E9%A3%9F%E6%94%BB%E7%95%A5/

### DISCUSSION THREAD

**Discussion by giscus** [beiyuouo/hainanu-course-comments#3] [General]
# hainanu-course-comments/1.%E5%AD%A6%E9%95%BF%E5%AD%A6%E5%A7%90%E8%AF%B4/%E6%B5%B7%E5%8D%97%E5%A4%A7%E5%AD%A6%E7%BE%8E%E9%A3%9F%E6%94%BB%E7%95%A5/

https://www.bj-yan.top/hainanu-course-comments/1.%E5%AD%A6%E9%95%BF%E5%AD%A6%E5%A7%90%E8%AF%B4/%E6%B5%B7%E5%8D%97%E5%A4%A7%E5%AD%A6%E7%BE%8E%E9%A3%9F%E6%94%BB%E7%95%A5/

**Comment by Iriszyh** [comment]
五食堂一楼，豆花饭的红烩的小酥肉 绝绝子！！
二食堂 自助旁边打饭的那里，烤鸭肉好吃（海大食堂最好吃的鸭肉)那个瘦瘦矮矮的阿姨打得比较多
七食堂一楼（北）在之前的轻食旁边的那家炒粉，no.1！（孩子吃过所有食堂的炒粉，这个真的好好吃！）第二好吃的炒粉是早上二食堂自助那里的
最喜欢五食堂一楼的粉汤

**Reply by yyyanbj** [reply]
好家伙，说吃的还是你懂得多:thumbsup:

**Reply by Iriszyh** [reply]
那肯定，而且这只是我这学期的最爱🌝🌝

**Comment by Iriszyh** [comment]
加一条！五食堂二楼网红炒饭都还不错，那个小咸菜巨好吃！魔饭青年的鱿鱼超好吃！，五食堂一楼的烤盘好便宜，二食堂的水果捞最新鲜！
避雷：二食堂麻辣烫（一食堂麻辣烫还可以），六食堂三楼的豚骨拉面（快跑！）

**Comment by Iriszyh** [comment]
还有学校附近的：
北门四川人家重庆人家yyds！
栖枝我个人觉得不好吃还很贵（你杠就是你对）
我觉得那家什么卷饼，宝真超市那的也不好吃
十八碗的紫苏牛蛙（好像是这个名字）还有手撕鸡巨好吃！！他家的糖油粑粑不太好吃
东门左拐十字路口那的党小旗超级超级超级好吃！我只吃了烧烤，超好吃！他家的炒粉也好吃！！
德慧轩炸炸好吃！！！！！！！（可以骑车过去，海甸城再走一个十字路口，我去都是打车过去的，来回不到20）德慧轩炸炸在后面，前面是德慧轩水吧，老盐水很好喝（我特别不喜欢喝老盐水的都能喝几口！）
东门外的那家巨大手作披萨（走着过去，不远）我觉得牛肉披萨➕红薯泥的底 绝绝子！
东门南门之间的那个单人火锅也好吃（两个人去吃也可以的嗷），小隔间可以升起来
京华城必胜客对面二楼那个什么什么餐厅（听起来有点高大上又有点土的那个），日料楼上那家，巨！难吃！服务也！特别！差！
总结：个人觉得披萨no.1，第二是烧烤，第三炸炸（npy觉得炸炸第一披萨第二

**Reply by QIN2DIM** [reply]
+ 南门杨国福麻辣烫

**Reply by yyyanbj** [reply]
俺觉得南门马玉涛好次:yum:

**Reply by yyyanbj** [reply]
四川人家 no, 重庆人家 yes！北门的烤鱼其实也不错，美团上才40多一点，2人份够吃了。北门外面靠东门那边有一家叫洪福锅包肉，墙裂推荐！小吃街强推天津那个阿姨的手抓饼，其他摊位的感觉都不如那家好吃

**Reply by Iriszyh** [reply]
嗷 南门京尚道，我觉得不好吃（至少石锅拌饭和部队锅不好吃，炸鸡还可以） 那家鲍汁黄焖鸡感觉挺好吃，肉很好吃，srds肉不多
南门小吃街中间那家长沙臭豆腐特别特别好吃！两个阿姨（？）姐姐（？）卖的那个，不知道咋描述（哈哈哈哈我觉得比长沙的好吃哈哈哈哈，我喜欢吃皮脆的那种）

**Reply by Iriszyh** [reply]
南门小吃街的那个五道口还是啥的（差不多这种意思）炒粉炒粉绝绝子，还有个叫小胖还是啥的（牌子底色是蓝色的）鸡柳好吃。这两家等我下次去看看叫啥，先占坑怕我忘了
南门右转那边的疆小胖新疆炒米粉！好吃！

**Reply by yyyanbj** [reply]
应该是南门小吃街海甸西里口，他家炒粉qs好吃，不过他家鸭头还是最牛的，炸鸭头很绝！

**Reply by Iriszyh** [reply]
对对对！就是这个名！那我下次尝试一下鸭头

**Comment by Sophia030** [comment]
北门出门不过天桥直走，有家洪福锅包肉，他们家的锅包肉真的是最好吃的！！

**Comment by visions-ma** [comment]
点名推荐艾伦汉堡，质量秒杀KFC、麦当劳等小汉堡，一个管饱，缺点就是配送很慢（通常一个半小时到两个小时），再就是晚上关店很早（七点半关店），饿了么或者微信搜索艾伦美食联盟公众号都可以点单，他家天天爆单。
再就是北门有家太二酸菜鱼，走高端路线，很好吃，我18年来海南第一次吃太二这家，真不错。
北门往东走有家俞姐椒麻鸡，是新疆的椒麻鸡，环境很小，小破地，但是真的很好吃，而且尝得出来是散养鸡（鸡骨头特别硬，肉质很好），也不贵，一个人20多一份套餐，饭是抓饭，特别香。
